### ISSUE TITLE
Angular 12 / 13 support

### ISSUE THREAD

**Issue by JohannesHoppe** [kauppfbi/ngx-deploy-docker#97]
The following error occurs on a fresh Angular 13 project:

```
ng add ngx-deploy-docker
ℹ Using package manager: npm
✔ Found compatible package version: ngx-deploy-docker@0.0.5.
✔ Package information loaded.

The package ngx-deploy-docker@0.0.5 will be installed and executed.
Would you like to proceed? Yes
✔ Package successfully installed.
An unhandled exception occurred: NOT SUPPORTED: keyword "id", use "$id" for schema ID
See "/private/var/folders/65/0k8dncr10pj8lqmh192hjqgc0000gn/T/ng-n0tXRR/angular-errors.log" for further details.
```

This is well-know breaking change of Angular 12, please see this commit:

https://github.com/angular-schule/ngx-deploy-starter/commit/55b7c98dd7e8094ec68db232aa1abd4a2b4650a8

We also added support for Angular 13 to the `npx-deploy-starter` demo here:

https://github.com/angular-schule/ngx-deploy-starter/commit/337e3c6d9edc22746528f2b2e9a33da7dbabd3d3

Greetings,
Johannes

**Comment by kauppfbi** [comment]
Hi @JohannesHoppe, 

Thank you for opening the issue and sharing the resources.

I yesterday released the new version 0.1.0 with support for Angular v13. I did not invest much time on supporting and testing older versions of Angular, but they probably work as well. 

Can you please verify on your site that it's working now?

**Comment by JohannesHoppe** [comment]
I scrolled through the commit - code looks fine. I did a quick `ng add ngx-deploy-docker` - no issues, too.
But I have nothing to deploy to docker, but I thinks this should work as expected! 👍
